src/libstd/io/stdio.rs

```diff
@@ -875,7 +875,7 @@ impl fmt::Debug for StderrLock<'_> {
 /// Equivalent to the [`input`] method except that a prompt
 /// is printed before capturing input. This function returns
 /// either success ([`Ok`]) or failure ([`Err`]) containing
-/// the input. See the [`std::result`] module documentation
+/// the input. See the [`io::result`] module documentation
```

Comment:
I don't think [`io::result`] is a module documentation, it probably is `io::Result`. Same for the one below.

Reply:
resolved doc links